src/test/scala/org/broadinstitute/dsde/firecloud/service/NamespaceApiServiceSpec.scala

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.core.AgoraPermissionHandler
+import org.broadinstitute.dsde.firecloud.model.MethodRepository.{ACLNames, AgoraPermission, FireCloudPermission}
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.broadinstitute.dsde.firecloud.model.UserInfo
+import org.broadinstitute.dsde.firecloud.webservice.NamespaceApiService
+import org.mockserver.model.HttpRequest._
+import spray.http.HttpMethods
+import spray.http.StatusCodes._
+import spray.httpx.SprayJsonSupport._
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+
+class NamespaceApiServiceSpec extends BaseServiceSpec with NamespaceApiService {
+
+  val namespaceServiceConstructor: (UserInfo) => NamespaceService = NamespaceService.constructor(app)
+
+  def actorRefFactory = system
+
+  val urls = List("/api/methods/namespace/permissions", "/api/configurations/namespace/permissions")
+
+  val agoraPermission = AgoraPermission(
+    user = Some("random@site.com"),
+    roles = Some(ACLNames.ListOwner)
+  )
+
+  val fcPermissions = List(AgoraPermissionHandler.toFireCloudPermission(agoraPermission))
+
+  "NamespaceApiService" - {
+
+    "when calling GET on a namespace permissions path" - {
+      "a valid list of FireCloud permissions is returned" in {
+        urls map {
+          url =>
+            Get(url) ~> dummyUserIdHeaders("1234") ~> sealRoute(namespaceRoutes) ~> check {
+              status should equal(OK)
+              val permission = responseAs[List[FireCloudPermission]]
+              permission shouldNot be (None)
+            }
```

Comment:
because you control the response inside MockAgoraDAO, you can - and I think should - make the test more strict:
```
            Get(url) ~> dummyUserIdHeaders("1234") ~> sealRoute(namespaceRoutes) ~> check {
              status should equal(OK)
              val permission = responseAs[List[FireCloudPermission]]
              permission should be (Seq(FireCloudPermission("test-user@broadinstitute.org", ACLNames.Owner)))
            }
```

you could also make the test values in MockAgoraDAO public and read them, instead of copying/hardcoding them here.